src/routes/propertyRoutes.js

```diff
@@ -8,20 +8,339 @@ import { verifyToken } from '../middleware/auth';
+ *          description: an authentication header
+ *          required: true
+ *          type: string
+ *      responses:
+ *        200:
+ *          example: {
+ *           "success": true,
+ *           "property": {
+ *              "id": 20,
+ *              "image_url": "flat.png",
+ *              "title": "2 bedroom flat",
+ *              "address": "20, Ijegun Road",
+ *              "land_area": 200sq/meter,
+ *              "no_of_rooms": 3,
+ *              "no_of_bedrooms": 4,
+ *              "no_of_garage": 1,
+ *              "no_of_store": 1,
+ *              "year_of_build": 2012,
+ *              "purpose": "sale"
+ *           }
+ *          }
+ *      404:
+ *        description: Property not found
+ *      500:
+ *        description: Internal server error
+ */
 
 propertyRouter.get('/agent/properties/:id', verifyToken, getAgentProperties);
 
+/**
+ * @swagger
+ * /v1/agent/properties/{id}:
+ *   put:
+ *      tags:
+ *        - Property modification
+ *      description: Modify an already added property information
+ *      produces:
+ *        - application/json
+ *      parameters:
+ *        - name: id
+ *          description: ID of the property
+ *          in: path
+ *          required: true
+ *          type: integer
+ *        - name: property
+ *          description: Property object with updated information
+ *          in: body
+ *          required: true
+ *          schema:
+ *            $ref: '#/definitions/Property'
+ *            type: object
+ *            required:
+ *              - propertyId
+ *            properties:
+ *              propertyId:
+ *                type: integer
+ *            example: {
+ *                propertyId: 10
+ *            }
+ *        - name: Authorization
+ *          in: header
+ *          description: an authentication header
+ *          required: true
+ *          type: string
+ *      responses:
+ *        200:
+ *          description: Property modified successfully
+ *          example: {
+ *           "property": {
+ *              "id": 20,
+ *              "image_url": "flat.png",
+ *              "title": "2 bedroom flat",
+ *              "address": "20, Ijegun Road",
+ *              "land_area": 200sq/meter,
+ *              "no_of_rooms": 3,
+ *              "no_of_bedrooms": 4,
+ *              "no_of_garage": 1,
+ *              "no_of_store": 1,
+ *              "year_of_build": 2012,
+ *              "purpose": "sale"
+ *           },
+ *            "message": "Property updated successfully!"
+ *       }
+ *        400:
+ *         description: All field are required
+ *        401:
+ *         description: Invalid token supplied
+ *        404:
+ *         description: Property not found
+ *        500:
+ *         description: Internal server error
+ */
 
 propertyRouter.put('/agent/properties/:id', verifyToken, editProperty);
 
+/**
+ * @swagger
+ * /v1/agent/properties/{id}:
+ *   delete:
+ *      tags:
+ *        - Delete property
```

Comment:
There is no need to have different tags for each endpoint, there should be one for User authentication and the other is for Property.